common/djangoapps/student/roles.py

```diff
@@ -219,6 +219,21 @@ def users_with_role(self):
         return entries
 
 
+@register_access_role
+class GlobalCourseCreatorRole(RoleBase):
+    """
+    A global course creator with access to all site's courses.
+    """
+    ROLE = 'global_course_creator'
+
+    def __init__(self, *args, **kwargs):
+        """
+        Args:
+            course_key (CourseKey)
```

Comment:
should not arg be org instead of course_key?

Reply:
changed